### PR TITLE
build a simple HTML list of To Do tasks

### PR DESCRIPTION
**Pull request by MMGGYY66**
# Project requirements

- Set up a new project with webpack      
- Create an index.html file and write HTML markup here. 
- Create an empty To-Do List placeholder.
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
        description [string].
        completed [bool].
        index: [number].
- On page load, render the dynamically created list of tasks in the dedicated placeholder.
- Create a style.css and set rules for the To-Do List

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Eldimardash, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe for this project adheres to the following standards:
- [ ] All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the FAQ section is optional.
- [ ]  The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/generatetodo.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function generatetodo() {
-  console.log("todo");
+  console.log("todo list");
 }
 
 export default generatetodo
```
Comment (julie-ify) [inline src/generatetodo.js]: - [ ] You should have a list of objects in an array, with each object containing these items description (string), completed (boolean), index (number), like this:

  ```
  const tasks = [
    {
      description: 'An example description in string',
      completed: false,
      index: 1,
    },
    {
      description: 'Another example description in string',
      completed: true,
      index: 2
    }
  ];
  ```

File: src/template.html [hunk 2]
```diff
@@ -14,6 +14,12 @@
...
     <h3>Today's to do:</h3>
     <div id="todo" class="todo"></div>
+          <label for="add-todo" id="add-todo-label">
+            <input type="text" name="add-todo" id="add-todo" placeholder="Add to your list..." />
+            <img id="enter" />
+          </label>
+          <ul id="list"></ul>
+
     <button id="todoBtn" class="btn">Clear all completed</button>
   </div>
```
Comment (julie-ify) [inline src/template.html]: - [ ] The display of tasks should be handled by javascript. This means you should have a method that iterates over the tasks above and render them in HTML.

- [ ] Kindly ensure your styling matches the wireframe from the video.

  Your Implementation | The Design
  :---:|:---:
  rueo | kdkd

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @MMGGYY66,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**